web/styles/subscriptions.css

```diff
@@ -414,70 +428,19 @@ h4.user_group_setting_subsection_title {
     }
 }
 
-#subscription_overlay .two-pane-settings-container {
-    container: subscriptions / inline-size;
-}
-
-/* There's an awkward width of modal that is wider than the
-   breakpoint that collapses the left pane, but still narrow
-   enough that the sort buttons need a second row to be visible
-   without overlapping other UI. Note that for 16px and larger
-   font sizes, we always show two rows of buttons.
-
-   We also show two rows of buttons on particularly narrow
-   screens where there's only one panel and the overlay doesn't
-   have enough space for all the buttons. */
-@container settings-overlay (
-    ((width >= $settings_overlay_sidebar_collapse_breakpoint) and (width < calc(80em + 80px)))
-    or ((width < $settings_overlay_sidebar_collapse_breakpoint) and (width <= 36em))
-) {
-    #subscription_overlay .two-pane-settings-container {
-        .left {
-            .list-toggler-container {
-                height: 5em;
-                justify-content: end;
-                flex-wrap: wrap;
-            }
-
-            .tab-switcher {
-                margin-right: 0;
-            }
-
-            .tab-switcher:first-child {
-                /* This forces the other buttons to the next row */
-                width: 100%;
-                display: flex;
-                justify-content: end;
-            }
-        }
-
-        .right .display-type {
-            height: 5em;
-        }
-
-        .streams-list {
-            /* Calculated from several heights and padding/margin measurements
-            in .list-toggler-container and .stream_filter, with some added
-            fiddling to see what worked best at 12px - 20px font sizes.
-            Notably this height is shorter than the similar calculation
-            below because the toggler is taller.
-            When we redesign this area we should make this less brittle. */
-            height: calc(100% - 5.4em - 40px);
-        }
-
-        #stream_settings {
-            height: calc(100% - 5em);
-        }
-    }
-}
-
 .two-pane-settings-container {
-    position: relative;
+    display: grid;
+    grid-template:
+        "header   " var(--settings-overlay-header-height)
```

Comment:
As we add more grids with named areas, it would probably be a good idea to make them more unique. `settings-overlay-*` is a good candidate prefix here, and it would be concordant with the variable name here, too.

Reply:
Ah great call, I hadn't considered that grid names could interfere with each other. Thanks, done!